docs/content/theory/operators_index.rst

```diff
@@ -0,0 +1,39 @@
+.. _operators_index:
+
+Operators
```

Comment:
In order to reduce the depth of the tree, I would be inclined to remove this index file and simply have Interpolation, Averaging and Differential Operators at top-level

Reply:
"Flattening" of the tree might come naturally when the new doc style is implemented by Joe. I wanted to leave some of the final details of this to the end because we aren't 100% sure what the final look of the website will be yet. The important thing for me is that we avoid lengthy pages and that we try to partition things into easily digestible bites.

Reply:
Another thing to keep in mind is that the theory section currently parallels with the tutorials section. If you flatten or rearrange the content of the theory sections, the tutorials no longer line up.